src/ipinfo.rs

```diff
@@ -303,6 +303,19 @@ impl IpInfo {
     /// }
     /// ```
     pub async fn lookup(&mut self, ip: &str) -> Result<IpDetails, IpError> {
+        self._lookup(ip, BASE_URL).await
+    }
+
+    // Lookup your own v6 IP
```

Comment:
Can this be made into a doc comment? With sufficient explanation of purpose and limitations.